### PR TITLE
CFY-7481 add user-groups system-role to cloudify documentation

### PR DESCRIPTION
**Pull request by yarivgiga**

### CODE REVIEW

File: content/manager/roles-management.md [hunk 1]
```diff
@@ -9,16 +9,16 @@ weight: 1400
 ## What are Cloudify roles?
 
 The Cloudify roles provide sets of permissions, and define the access a user has in the context of tenants.
-Each user can get a role in a few ways – specifically assigned (per tenant),
-assigned to group (per tenant) or system level role when the user is created.
+Each user can get a role in a few ways – specifically assigned or assigned to group.
+A role can be a system-wide-role - which is valid in the whole Cloudify Manager, or a tenant-role, which only applies in a specific tenant.
```
Comment (dusking) [inline content/manager/roles-management.md]: if we used system-wide-role, we should use tenant-*wide-*role

File: content/manager/roles-management.md [hunk 2]
```diff
@@ -9,16 +9,16 @@ weight: 1400
...
 
 ## System-wide roles and tenant-roles
 
 System-wide roles are roles that define the access to all the tenants in the system,
 typically allowed to do system wide operations.
 Tenant roles are roles that define the access in the context of a specific tenant.
-User can have one system-wide role, but multiple roles per tenant.
-When a user or a group assigned to a tenant, it must assigned with a specific role.
+User can have multiple system-wide roles, as well as multiple roles per tenant (The multiple roles quality is possible using groups).
```
Comment (dusking) [inline content/manager/roles-management.md]: roles per tenant => tenant-wide roles
what is "multiple roles quality"?

File: content/manager/roles-management.md [hunk 3]
```diff
@@ -8,35 +8,24 @@ weight: 1400
...
-
-Tenant’s Role:
-
-- `manager` has the ability of managing all of the tenant's resources (private and public) and creating new resources,
-  but without the ability of creating or managing users.
-- `user` has the ability of managing all of the tenant's resources (only public) and creating new resources.
-- `operations` has the ability of deploying blueprints and executing workflows,
-  but does not allow them to upload new blueprints or plugins to the tenant.
-- `viewer` has the ability of viewing the public resources of explicitly defined tenant (or tenants).
+System roles apply the permissions in the role to the user for all of the tenants.
```
Comment (yarivgiga) [inline content/manager/roles-management.md]: and also outside of tenants

File: content/manager/roles-management.md [hunk 4]
```diff
@@ -8,35 +8,24 @@ weight: 1400
 
 ## What are Cloudify roles?
 
-The Cloudify roles provide sets of permissions, and define the access a user has in the context of tenants.
-Each user can get a role in a few ways – specifically assigned (per tenant),
-assigned to group (per tenant) or system level role when the user is created.
+A role is a group of permissions that are required by a certain type of user to work in Cloudify. You can assign roles to a user to give that user the permissions that are defined in the role. You can also assign roles to user groups to give the permissions that are defined in the role to all of the users in the group.
+If a user is a member of more than one group, then the user has all of the permissions defined for the user specifically, in addition to all of the permissions defined for all of the roles the user is assigned to.
```
Comment (yarivgiga) [inline content/manager/roles-management.md]: (both directly and/or via groups)

**Pull request merged**